src/EStimLibrary/Extensions/SpatialModel/StringHierarchy/StringHierarchySpec.cs

```diff
@@ -81,118 +117,70 @@ public static string JoinFullSpec(string[] regionSet, string[] modifierSet)
             regionSpec;
     }
 
+    /// <summary>
+    /// Join a sequence of optioned region string names into a single string,
+    /// separated by the REGIONS_DELIMITER.
+    /// Inverse of ParseRegionSpec, although no case conversion or trimming.
+    /// </summary>
+    /// <param name="regionSet">The ordered array of optioned region string 
+    /// names.</param>
+    /// <returns>The single joined string region spec.</returns>
     public static string JoinRegionSet(string[] regionSet)
     {
         return string.Join(REGIONS_DELIMITER, regionSet);
     }
 
+    /// <summary>
+    /// Join a set of directional modifiers into a single string modifier spec,
+    /// separated by the MODIFIERS_DELIMITER.
+    /// Inverse of ParseModifierSpec, although no case conversion or trimming.
+    /// </summary>
+    /// <param name="modifierSet">The string array of unique modifiers.</param>
+    /// <returns>The single join string modifier spec.</returns>
     public static string JoinModifierSet(string[] modifierSet)
     {
         return string.Join(MODIFIERS_DELIMITER, modifierSet);
     }
 
     /// <summary>
     /// Parse the region option - if there is one - and base name from a full
-    /// region name.
+    /// optioned region name.
     /// </summary>
-    /// <param name="optionedRegionName">The full region name to parse.</param>
-    /// <param name="baseName">An output parameter: the base name upon success,
-    /// else and empty string.</param>
-    /// <param name="options">An output parameter: the string of ordered options
-    /// correctly delimited if there are any, empty if not.</param>
+    /// <param name="optionedRegionName">The full optioned region name to parse.
+    /// </param>
+    /// <param name="baseName">An output parameter: the whitesapce trimmed base 
+    /// name upon success, else an empty string.</param>
+    /// <param name="option">An output parameter: the whitespace trimmed option 
+    /// name if there is any, else an empty string.</param>
     /// <returns>True if valid parse, False if not.</returns>
     public static bool TryParseOptionedRegionName(string optionedRegionName,
-        out string baseName, out string options)
+        out string baseName, out string option)
```

Comment:
This should be "options", e.g., "left index finger" should be valid where "finger" is the region basename and "left index" is the option string for that region. Forgive me if I made these changes making it now incorrect, but (1) it should be reverted back to what is in red. (2) Add a test case or so for this case. (3) Write a method in StringHierarchySpec called TryParseOptions() that uses the OPTIONS_REGION_DELIMITER to split an "options" string as output by TryParseOptionedRegionName, returning a bool (true if any options found, else false) and outputting (via an `out` param) the string[] of option names. Should handle an empty, single option, and multi-option-containing string. Should also trim whitespace from each option name found. (4) Write test cases for that function too.